elements/pfe-card/test/pfe-card.spec.ts

```diff
@@ -2,6 +2,8 @@ import { expect, html, aTimeout } from '@open-wc/testing';
 import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
 import { hexToRgb, getColor } from '@patternfly/pfe-tools/test/hex-to-rgb.js';
 
+import '@patternfly/pfe-tools/test/stub-logger.js';
```

Comment:
Same as the comment on pfe-band, not sure if this is doing something I'm not recognizing here but it doesn't look like the component itself logs or uses the logger class